Project/Source/Components/ComponentAgent.cpp

```diff
@@ -132,6 +132,18 @@ void ComponentAgent::RemoveAgentFromCrowd() {
 	agentId = -1;
 }
 
+float3 ComponentAgent::GetVelocity() const {
```

Comment:
Why are you using this new method instead of the already existing GetMaxVelocity?